kythe/cxx/common/status_or.h

```diff
@@ -58,34 +58,58 @@ class ABSL_MUST_USE_RESULT StatusOr final {
   template <typename U>
   StatusOr& operator=(StatusOr<U>&& other);
 
+  /// \brief Return this->status().ok()
```

Comment:
It returns this->status_.ok()

Reply:
Which is equivalent; I would far rather documentation refer to public members of a class than private members.

Reply:
It's weird for me to see the implementation of a thing copied inexactly in a documentation comment. Can you describe what the function does in an alternate way?

Reply:
Sure, this is largely taken from the internal equivalent, though :-)